src/store/reducers/contracts.reducer.ts

```diff
@@ -46,11 +49,23 @@ export const contractsSlice = createSlice({
           : item,
       );
     },
+    updateFilter: (state, action) => {
+      state.filter = action.payload;
+    },
     updateFeedback: (state, action) => {
       state.offers = state.offers.map(item =>
         item.id === action.payload.id ? { ...item, org_feedback: action.payload.orgFeedback } : item,
       );
     },
+    updatePage: (state, action) => {
+      state.page = action.payload;
+    },
+    handleOpenSlider: state => {
```

Comment:
why did you put 2 functions for one functionality? you can pass open option as `payload`